schema/meta.go

```diff
@@ -5,11 +5,10 @@ import (
 	"time"
 
 	"github.com/google/uuid"
-	"github.com/rs/zerolog"
 )
 
 type ClientMeta interface {
-	Logger() *zerolog.Logger
+	Name() string
```

Comment:
```suggestion
	ID() string
```

Not sure but this should be unique otherwise the log message in `resolveTableDfs` won't make sense correct?
https://github.com/cloudquery/plugin-sdk/pull/318/files#diff-9ba9bcdf8d30be3df89fb7ad1cffbaa5cdd72b28097d7eec3cd1eb5f5dea5e7bR70

Also what's the reason for the removal of `Logger`?

> Changes to `ClientMeta` are technically breaking changes (which is OK).
If we want we can make this a non breaking change by using properties (e.g. `Name string`) and then plugins need to add them via the constructor otherwise the log field will be empty

Reply:
We don't use the logger and we don't need to enforce it on the interface level. the only thing we need is `ID` or `Name` so logs will be easily understood.

Reply:
changed to `ID`